examples/resources/configstore_entries_basic_usage_managed_entries.tf

```
@@ -0,0 +1,39 @@
+# IMPORTANT: Deleting a Config Store requires first deleting its resource_link.
+# This requires a two-step `terraform apply` as we can't guarantee deletion order.
+# e.g. resource_link deletion within fastly_service_compute might not finish first.
+resource "fastly_configstore" "example" {
+  name = "%s"
+}
+
+resource "fastly_configstore_entries" "example" {
+  store_id = fastly_configstore.example.id
+  entries = {
+    key1 : "value1"
+    key2 : "value2"
+  }
+  managed_entries = true
```

Should be `manage_entries`

```suggestion
  manage_entries = true
```